x-pack/plugins/cases/docs/openapi/paths/api@cases@{caseid}@comments.yaml

```
@@ -0,0 +1,94 @@
+post:
+  description: >
```

While working on https://github.com/elastic/kibana/pull/133840 today, I discovered there's a summary property that is "A short summary of what the operation does". I've added it to the rest of the APIs in my PR, so I think it's worthwhile using for these ones here too.
```suggestion
  summary: Adds a comment to a case.
  description: >
```